libraries/engage/page/widgets/ImageSlider/hooks.js

```diff
@@ -55,8 +55,12 @@ export const useImageSliderWidget = () => {
     slidesPerViewCustomMedium,
     slidesPerViewCustomLarge,
     imageSpacing,
+    paginationStyle = 'bullets',
   } = config;
 
+  const paginationType = useMemo(() => (paginationStyle === 'default' ? 'bullets' : paginationStyle.toLowerCase()),
```

Comment:
the values in the widget specification needs to be updated: progressBar => progressbar, Fraction => fraction